### ISSUE TITLE
Use formattedAmount in more places

### ISSUE THREAD

**Issue by sebastiaanspeck** [RocketC31/budget#93]
Right now, we use formattedAmount only in the mostExpensiveTags, but it's off with the other widgets. In the Netherlands (and probably as well in France), we use "," as separator when using a double.

![image](https://user-images.githubusercontent.com/12570668/203482050-8f652c45-c149-436f-af43-994225231daa.png)

Also maybe check how we want to co-op with this separator issue when creating a transaction/budget and entering an amount. The regex also fails if you enter 10.5, this needs to be resolved to 10.50 as amount (maybe create a new issue for this).

**Comment by RocketC31** [comment]
True constatation ! I'm too frustrating by this but i have never make time to correct it but yes good issue !
I think the two one can be make on the same issue no ? Globaly the problem is the formatting so if we fix the `,` and the 0 to add if we send `10.5` it's good no ?

For that it's like you prefer

**Comment by sebastiaanspeck** [comment]
Yes, it could be fixed if we fix the second issue, but it are two different places. One is in Vue and the other in PHP.Met vriendelijke groet,Sebastiaan SpeckOn 23 Nov 2022, at 07:27, RocketC31 ***@***.***> wrote:﻿
True constatation ! I'm too frustrating by this but i have never make time to correct it but yes good issue !
I think the two one can be make on the same issue no ? Globaly the problem is the formatting so if we fix the , and the 0 to add if we send 10.5 it's good no ?
For that it's like you prefer

**Comment by RocketC31** [comment]
It does not shock me then if it is a part in front, another in back
